apps/blog-app/src/styles.css

```
@@ -85,36 +235,35 @@ img:not([width]):not([height]) {
   /* Inline code */
   code {
-    background-color: #f5f5f5;
+    background-color: rgb(var(--code-bg));
+    color: rgb(var(--code-fg));
     padding: 0.2rem 0.4rem;
-    border-radius: 3px;
-    font-size: 1rem;
-    font-family: 'Courier New', Courier, monospace;
+    border-radius: 4px;
+    font-size: 0.95rem;
+    font-family: 'SFMono-Regular', ui-monospace, 'Courier New', Courier, monospace;
```

Stylelint (17.12.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)